test.py

```diff
@@ -4,8 +4,10 @@
 import unittest
 
 class MathTestCase(unittest.TestCase):
-    def verify(self, mkd_name, html_name, enable_dollar_delimiter=False):
-        config = {'enable_dollar_delimiter': enable_dollar_delimiter}
+    def verify(self, mkd_name, html_name, enable_dollar_delimiter=False,
+            render_to_span=False):
+        config = {'enable_dollar_delimiter': enable_dollar_delimiter,
```

Comment:
`enable_dollar_delimiter` is actually used in one of the tests. I don't mind if you test your option too, but if you don't, please remove the useless argument.